.NET/Microsoft.Recognizers.Text.Sequence/Extractors/BasePhoneNumberExtractor.cs

```diff
@@ -130,8 +130,11 @@ public override List<ExtractResult> Extract(string text)
                             var charGap = text[(int)(er.Start - 2)];
                             if (!char.IsNumber(charGap) && !char.IsWhiteSpace(charGap))
                             {
+                                // check if the extracted string has a non-digit string before "-".
+                                var flag = Regex.IsMatch(text.Substring(0, (int)(er.Start - 2)), @"^[^0-9]+$");
```

Comment:
Checking some kind of prefix for the tagged potential phonenumber only if fortmatted? Is that it?

Reply:
yes